README.md

```diff
@@ -2,10 +2,56 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+It is the mission of the Node.js Foundation and the TSC to provide a home for 
+openly-governed and community-managed open source projects that are focused on 
+supporting and providing benefit to the Node.js ecosystem.
```

Comment:
To "enable and encourage the adoption" isn't actionable enough. The entire foundation is doing that, the question is "how is the TSC doing that?" and specifically "what is the TSC doing outside of hosting Node.js core to do that?"

The "neutral home for owning assets" part of this is becoming important to some projects. It's something that we needed when we started the foundation, and you can see similar needs popping up in the community. I don't think that we can debate whether or not there are projects that need that, it's clear they do because they are asking for it, all we can debate is whether or not that home should be under the TSC. The purpose of this PR is to have that debate.